src/components/app/App.tsx

```diff
@@ -22,7 +22,7 @@ export class Evergreen extends React.Component<Props, State> {
   constructor(props: Props) {
     super(props);
     this.state = {
-      APIClient: null,
+      APIClient: rest.EvergreenClient("", "", "", "", true),
```

Comment:
We can leave this as null can't we? Assuming that you upload a config file before going to a page that needs to use the client

Reply:
Yes you can, the only problem is that the patches page depends on that client existing and being not null in order to load all the patches. I switched it back to an empty client because it was really annoying to have to upload a config file every time I wanted to test something on the patches page. I can make it null again if you think that's better, it was just inconvenient when testing.

Reply:
Fair, let's leave it like this. I was annoyed by having to upload it each time as well